src/themes/elife/lib/query.ts

```diff
@@ -7,6 +7,7 @@ export interface articleData {
   pdf: string
   figuresPdf?: string
   copyright: { license: string }
+  type: 'replication-study' | 'research-article'
```

Comment:
We need to confirm all of the article types for which we may have an ERA companion.